mlflow/models/utils.py

```diff
@@ -1508,3 +1516,99 @@ def _convert_llm_input_data(data):
             data = data.to_dict(orient="records")
 
     return _convert_llm_ndarray_to_list(data)
+
+
```

Comment:
@annzhang-db To make these PRs easier to review in the future, can you leave inline comments indicating which functions are copied verbatim from one place to another? This is massively helpful for reviewers :D